spec.md

```diff
@@ -450,6 +450,7 @@ It is therefore RECOMMENDED that implementations avoid such strings.
 | bindable* | boolean | Specifies whether Service Instances of the service can be bound to applications. This specifies the default for all Service Plans of this Service Offering. Service Plans can override this field (see [Service Plan Object](#service-plan-object)). |
 | instances_retrievable | boolean | Specifies whether the [Fetching a Service Instance](#fetching-a-service-instance) endpoint is supported for all Service Plans. |
 | bindings_retrievable | boolean | Specifies whether the [Fetching a Service Binding](#fetching-a-service-binding) endpoint is supported for all Service Plans. |
+| allow_context_updates | boolean | Specifies whether a Service Instance supports [Update](#updating-a-service-instance) requests when contextual data for the Service Instance in the Platform changes. |
```

Comment:
`allow_context_updates` -> `context_updateable` to be consistent with existing flags like `bindings_retrievable` ?

Reply:
Can the context object also change during a normal instance update?

Reply:
Yes, I think you can infer that from the spec today.

The aim with this flag is to allow service brokers to specify whether they would like to receive update requests when *only* context data changes.